WordPress/WordPressTest/Dashboard/FreeToPaidPlansDashboardCardHelperTests.swift

```diff
@@ -0,0 +1,69 @@
+import XCTest
+@testable import WordPress
+
+final class FreeToPaidPlansDashboardCardHelperTests: CoreDataTestCase {
+    func testShouldShowCardWithFreePlanAndNoMappedDomain() {
+        let blog = BlogBuilder(mainContext)
+            .with(supportsDomains: true)
+            .with(domainCount: 1, of: .wpCom)
+            .with(hasMappedDomain: false)
+            .with(hasPaidPlan: false)
+            .build()
+
+        let result = FreeToPaidPlansDashboardCardHelper.shouldShowCard(for: blog, isJetpack: true, featureFlagEnabled: true)
+
+        XCTAssertTrue(result, "Card should show for blogs with a free plan and a mapped domain")
```

Comment:
```suggestion
        XCTAssertTrue(result, "Card should show for blogs with a free plan and no mapped domain")
```

This might be a typo. A site with a [mapped domain](https://github.com/wordpress-mobile/WordPressKit-iOS/blob/c323bc92ef806679bf830f87383d343f0f2f6469/WordPressKit/RemoteDomain.swift#LL18C1-L18C1) already has a domain, so I don't think we're offering the domain there.